src/views/v2/commerce/checkout/cart/item.php

```diff
@@ -35,6 +35,18 @@
 	return;
 }
 
+/**
+ * Filters whether the current item should be skipped in the checkout.
+ *
+ * @since TBD
+ *
+ * @param bool  $should_skip Whether the item should be skipped or not.
+ * @param array $item        The item to be checked.
+ */
+if ( apply_filters( 'tec_tickets_checkout_should_skip_item', false, $item ) ) {
```

Comment:
Why wait until the last minute in the template to filter this? Add a filtered prop to the item.

Reply:
Also - check the comment above the use - it's on the wrong line. (L31)

Reply:
> Add a filtered prop to the item.

@Camwyn Could be a little more specific about what you mean here? Or perhaps an example of where would be best to do that?

Reply:
When we assemble `$items` for the template, add a `tec_tickets_checkout_should_skip_item` to each item and filter it there, rather than here in the template.

Filters like this don't belong in a template. The template should just check it (as a passed "prop") and bail if needed.